有効化されているべきfeature `serde/derive`が有効化されていない

## 不具合の内容

#239 をやっているときに気づいたのですが、`voicevox_core`のCargo.tomlではfeature `serde/derive`が有効化されておらず、`dev-dependencies`によりたまたま有効化されているという状態です。

<https://github.com/VOICEVOX/voicevox_core/blob/75732e9575120dd7492bbc40879417417931cfea/crates/voicevox_core/Cargo.toml#L23>

これで何が起きるのかというと、crates/下に`voicevox_core`に依存する別のクレートを追加したときに、そのクレートのコンパイルが失敗します。

### 現象・ログ

<details>
<summary><code>dev-dependencies</code>抜きで<code>voicevox_core</code>のビルドを試みる</summary>

```console
❯ cargo hack check --no-dev-deps
info: --no-dev-deps removes dev-dependencies from real `Cargo.toml` while cargo-hack is running and restores it when finished
info: running `cargo check` on voicevox_core (1/1)
   Compiling voicevox_core v0.1.0 (/home/ryo/src/github.com/VOICEVOX/voicevox_core/crates/voicevox_core)
   Compiling open_jtalk-sys v0.16.111 (https://github.com/VOICEVOX/open_jtalk-rs.git#c77112b4)
   Compiling onnxruntime-sys v0.0.24 (https://github.com/VOICEVOX/onnxruntime-rs.git#8b45a4a9)
    Checking open_jtalk v0.1.25 (https://github.com/VOICEVOX/open_jtalk-rs.git#c77112b4)
    Checking onnxruntime v0.0.29 (https://github.com/VOICEVOX/onnxruntime-rs.git#8b45a4a9)
error: cannot find derive macro `Deserialize` in this scope
  --> crates/voicevox_core/src/status.rs:42:10
   |
42 | #[derive(Deserialize, Getters)]
   |          ^^^^^^^^^^^
   |
note: `Deserialize` is imported here, but it is only a trait, without a derive macro
  --> crates/voicevox_core/src/status.rs:8:13
   |
8  | use serde::{Deserialize, Serialize};
   |             ^^^^^^^^^^^

error: cannot find derive macro `Deserialize` in this scope
  --> crates/voicevox_core/src/status.rs:47:10
   |
47 | #[derive(Deserialize, Getters)]
   |          ^^^^^^^^^^^
   |
note: `Deserialize` is imported here, but it is only a trait, without a derive macro
  --> crates/voicevox_core/src/status.rs:8:13
   |
8  | use serde::{Deserialize, Serialize};
   |             ^^^^^^^^^^^

error: cannot find derive macro `Serialize` in this scope
  --> crates/voicevox_core/src/status.rs:66:26
   |
66 | #[derive(Getters, Debug, Serialize, Deserialize)]
   |                          ^^^^^^^^^
   |
note: `Serialize` is imported here, but it is only a trait, without a derive macro
  --> crates/voicevox_core/src/status.rs:8:26
   |
8  | use serde::{Deserialize, Serialize};
   |                          ^^^^^^^^^

error: cannot find derive macro `Deserialize` in this scope
  --> crates/voicevox_core/src/status.rs:66:37
   |
66 | #[derive(Getters, Debug, Serialize, Deserialize)]
   |                                     ^^^^^^^^^^^
   |
note: `Deserialize` is imported here, but it is only a trait, without a derive macro
  --> crates/voicevox_core/src/status.rs:8:13
   |
8  | use serde::{Deserialize, Serialize};
   |             ^^^^^^^^^^^

error[E0277]: the trait bound `status::SupportedDevices: Serialize` is not satisfied
    --> crates/voicevox_core/src/internal.rs:479:31
     |
479  |         serde_json::to_string(&SupportedDevices::get_supported_devices().unwrap()).unwrap(),
     |         --------------------- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Serialize` is not implemented for `status::SupportedDevices`
     |         |
     |         required by a bound introduced by this call
     |
     = help: the following other types implement trait `Serialize`:
               &'a T
               &'a mut T
               ()
               (T0, T1)
               (T0, T1, T2)
               (T0, T1, T2, T3)
               (T0, T1, T2, T3, T4)
               (T0, T1, T2, T3, T4, T5)
             and 127 others
note: required by a bound in `serde_json::to_string`
    --> /home/ryo/.cargo/registry/src/github.com-1ecc6299db9ec823/serde_json-1.0.81/src/ser.rs:2225:17
     |
2225 |     T: ?Sized + Serialize,
     |                 ^^^^^^^^^ required by this bound in `serde_json::to_string`

error[E0277]: the trait bound `Meta: Deserialize<'_>` is not satisfied
    --> crates/voicevox_core/src/status.rs:124:13
     |
124  |             serde_json::from_str(Self::METAS_STR).map_err(|e| Error::LoadMetas(e.into()))?;
     |             ^^^^^^^^^^^^^^^^^^^^ the trait `Deserialize<'_>` is not implemented for `Meta`
     |
     = help: the following other types implement trait `Deserialize<'de>`:
               &'a Path
               &'a [u8]
               &'a str
               ()
               (T0, T1)
               (T0, T1, T2)
               (T0, T1, T2, T3)
               (T0, T1, T2, T3, T4)
             and 135 others
     = note: required because of the requirements on the impl of `Deserialize<'_>` for `Vec<Meta>`
note: required by a bound in `serde_json::from_str`
    --> /home/ryo/.cargo/registry/src/github.com-1ecc6299db9ec823/serde_json-1.0.81/src/de.rs:2611:8
     |
2611 |     T: de::Deserialize<'a>,
     |        ^^^^^^^^^^^^^^^^^^^ required by this bound in `serde_json::from_str`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `voicevox_core` due to 6 previous errors
error: process didn't exit successfully: `/home/ryo/.rustup/toolchains/1.63.0-x86_64-unknown-linux-gnu/bin/cargo check --manifest-path Cargo.toml` (exit status: 101)
```

</details>


### 再現手順

↑のように`dev-dependencies`抜きで`voicevox_core`をビルドしようとすると、失敗する。

### 期待動作

`dev-dependencies`抜きでもコンパイルできる。

## OSの種類/ディストリ/バージョン

<!--
なるべく詳しく書いてください 記述例:
*   Windows 10 Pro 64bit (10.0.10586)
*   macOS Sierra
*   Linux fedora 23 64bit
*   Others
-->

## その他

<!-- 関連して何か気がついたこと、気になることがあればココに書いてください -->

--- a/crates/voicevox_core/Cargo.toml
+++ b/crates/voicevox_core/Cargo.toml
@@ -20,7 +20,7 @@ derive-new = "0.5.9"
 libc = "0.2.131"
 once_cell = "1.10.0"
 onnxruntime = { git = "https://github.com/VOICEVOX/onnxruntime-rs.git", version = "0.0.30" }
-serde = "1.0.143"
+serde = { version = "1.0.143", features = ["derive"] }
 serde_json = "1.0.83"
 thiserror = "1.0.32"
 open_jtalk = { git = "https://github.com/VOICEVOX/open_jtalk-rs.git", version = "0.1.25" }

